pkgs/by-name/ne/nexusmods-app/package.nix

```diff
@@ -58,7 +58,7 @@ buildDotnetModule rec {
 
   makeWrapperArgs = [
     "--prefix PATH : ${lib.makeBinPath [ desktop-file-utils ]}"
-    "--set APPIMAGE $out/bin/${meta.mainProgram}" # Make associating with nxm links work on Linux
+    "--set APPIMAGE ${placeholder "out"}/bin/${meta.mainProgram}" # Make associating with nxm links work on Linux
```

Comment:
What's the advantage of `${placeholder "out"}` over `$out`?

Reply:
Just using `"$out"` didn't work correctly; it ends up being included literally in the wrapper.

This was taken from feedback on one of the other PRs.

Reply:
:open_mouth: OK then, my bad! It (that is, how this was done *some time ago*) *used* to work, but I must've messed up.